app/src/main/java/org/dharmaseed/android/DataFetcherTask.java

```diff
@@ -76,7 +76,7 @@ protected void updateTable(String tableName, String tableID, String apiUrl, Stri
             builder.addFormDataPart("edition", edition);
         }
         Request request = new Request.Builder()
-                .url("http://www.dharmaseed.org/api/1/"+apiUrl)
+                .url("https://www.dharmaseed.org/api/1/"+apiUrl)
```

Comment:
@ericm108 or @ewoudenberg, just a heads up, I've changed all the dharmaseed API URLs in the app to use HTTPS instead of HTTP. This change is in response to this message I got on the Google Play console recently:

![2023-09-16-163156_1001x444_scrot](https://github.com/dharmaseed/dharmaseed-android/assets/10068296/c38c4c7c-1632-4528-ae30-a9bc529b4f55)

I have a vague recollection that perhaps the dharmaseed APIs were once only served over HTTP (is that correct?), which as far as I recall is why these URLs were using HTTP previously. In any case, the app seems to work fine using only HTTPS, so switching will make Google happy and avoid any potential adverse visibility changes in the Play Store. I wanted to mention this to you just to make sure you think it's OK to make this change.